src/main/java/chessmaster/storage/Storage.java

```diff
@@ -110,6 +110,11 @@ public void resetBoard() throws ChessMasterException {
     public ChessTile[][] loadBoard() throws ChessMasterException {
         createChessMasterFile();
 
+        int blackPieces = 0;
+        int whitePieces = 0;
+        boolean blackKing = false;
+        boolean whiteKing = false;
```

Comment:
Consider renaming the variables for readability:
- `blackPieceNum`, `whitePieceNum`
- `isBlackKingPresent`, `isWhiteKingPresent`